samples/peripheral/802154_phy_test/src/rf_proc.c

```diff
@@ -133,7 +139,26 @@ void rf_uninit(void)
 #if CONFIG_PTT_ANTENNA_DIVERSITY
 static void configure_antenna_diversity(void)
 {
-	/* TODO: Implement when antenna diversity is supported in NCS */
+	nrf_ppi_channel_t ppi_channel;
```

Comment:
Would be nice to move this code to generic sdk-nrf modules, like initialization of FEMs is handled now in subsys/mpsl/fem. But it's obviously out of scope of this PR.